source/templates/application_template.html.md.erb

```diff
@@ -136,24 +139,44 @@ gds govuk connect ssh -e production aws/<%= application.aws_puppet_class %>
 </ul>
 <% end %>
 
-<% if manual.pages_for_application(application.app_name).any? %>
-<h3>Relevant manual pages</h3>
+<% if application.example_rendered_pages %>
+## Example pages rendered by <%= application.app_name %>
 
 <ul>
-<% manual.pages_for_application(application.app_name).each do |page| %>
-  <li><%= link_to page.data.title, page.url %></li>
+<% application.example_rendered_pages.each do |example| %>
+  <li><%= link_to example['title'], "https://www.gov.uk#{example['link']}" %></li>
 <% end %>
 </ul>
 <% end %>
 
-<% if application.example_rendered_pages %>
-<h3>Example pages rendered by <%= application.app_name %></h3>
+## GitHub
 
-<ul>
-<% application.example_rendered_pages.each do |example| %>
-  <li><%= link_to example['title'], "https://www.gov.uk#{example['link']}" %></li>
+<%= link_to "github.com/alphagov/#{application.github_repo_name}", application.repo_url %>
+
+<% unless application.topics.nil? %>
+### Topics
```

Comment:
Agreed - I only kept it in case I was missing some historical context and others find it useful. I can't imagine it being particularly useful! Will remove.